vanilla-framework/vanilla-framework
# Dockerfile for docs

This should be merged after #1493 

## Done

Add a docker image for the docs app

## QA

- Pull code
- Install latest documentation-builder (`snap install documentation-builder`)
- `yarn run build-docs`
- `cd docs`
- `docker build . -t vanilla-docs --build-arg COMMIT_ID=$(git rev-parse HEAD)`
- `docker run --rm -p 8080:80 vanilla-docs`
- Visit http://localhost:8080/ and it should look beautiful!
